### ISSUE TITLE
Unable to initialize LeaderReplicaDistributionGoal

### ISSUE THREAD

**Issue by nevins-jask** [linkedin/cruise-control#711]
Seeing this error when attempting to start 2.0.44 with the default config
```
Exception in thread "main" org.apache.kafka.common.KafkaException: Could not find a public no-argument constructor for com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderReplicaDistributionGoal
	at org.apache.kafka.common.utils.Utils.newInstance(Utils.java:308)
	at org.apache.kafka.common.utils.Utils.newInstance(Utils.java:322)
	at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:355)
	at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:332)
	at com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig.getConfiguredInstances(KafkaCruiseControlConfig.java:1322)
	at org.apache.kafka.common.config.AbstractConfig.getConfiguredInstances(AbstractConfig.java:319)
	at com.linkedin.kafka.cruisecontrol.config.KafkaCruiseControlConfig.getConfiguredInstances(KafkaCruiseControlConfig.java:1311)
	at com.linkedin.kafka.cruisecontrol.analyzer.AnalyzerUtils.getGoalMapByPriority(AnalyzerUtils.java:152)
	at com.linkedin.kafka.cruisecontrol.monitor.LoadMonitor.<init>(LoadMonitor.java:175)
	at com.linkedin.kafka.cruisecontrol.monitor.LoadMonitor.<init>(LoadMonitor.java:127)
	at com.linkedin.kafka.cruisecontrol.KafkaCruiseControl.<init>(KafkaCruiseControl.java:118)
	at com.linkedin.kafka.cruisecontrol.async.AsyncKafkaCruiseControl.<init>(AsyncKafkaCruiseControl.java:72)
	at com.linkedin.kafka.cruisecontrol.KafkaCruiseControlMain.main(KafkaCruiseControlMain.java:71)
Caused by: java.lang.NoSuchMethodException: com.linkedin.kafka.cruisecontrol.analyzer.goals.LeaderReplicaDistributionGoal.<init>()
	at java.base/java.lang.Class.getConstructor0(Class.java:3350)
	at java.base/java.lang.Class.getDeclaredConstructor(Class.java:2554)
	at org.apache.kafka.common.utils.Utils.newInstance(Utils.java:306)
	... 12 more
```

**Comment by kidkun** [comment]
Thanks for reporting the issue. Will apply a fix today.

**Comment by efeg** [comment]
@nevins-jask the version with the fix is available at `2.0.45` -- feel free to reopen this issue if you see any relevant errors.
